src/ck-ldb/LBManager.C

```diff
@@ -694,9 +715,24 @@ void LBManager::nextLoadbalancer(int seq)
 }
 
 // switch strategy
-void LBManager::switchLoadbalancer(int switchFrom, int switchTo)
-{
-  // TODO: Implement turn off / on
+void LBManager::switchLoadbalancer(int switchFrom, int switchTo) {
```

Comment:
Why do we need switchFrom? Is this meant to be exposed to the application? If so, how would the application know the ints to pass? Could it use strings instead?

Reply:
It's needed to support non-TreeLBs. This function is used by metabalancer to switch to newly predicted balancers. When switching between two different LBs (note that switching between different strategies of TreeLB, it's not needed since they're both part of the same LB), it has to call `turnOff` and `turnOn`to do setup/cleanup work. So this is lying in wait for a more complete implementation of the metabalancer stuff. Metabalancer should know the ints, it could use strings, but it doesn't need to since this is internal stuff, anyway.

Reply:
#2818